minetest/snapcraft.yaml

```diff
@@ -15,13 +15,14 @@ apps:
 
 parts:
   launcher:
-    plugin: copy
-    files:
+    plugin: dump
+    source: .
+    organize:
       minetest: minetest
   minetestgame:
     source: https://github.com/minetest/minetest_game.git
-    plugin: copy
-    files:
+    plugin: dump
+    organize:
```

Comment:
Mind puting extra care on that one, to ensure that this organize with wildcards work? It wasn't before…